JSDemos/Demos/List/ListSelection/description.md

```diff
@@ -1 +1,9 @@
-The **List** UI component offers three item selection modes: _&laquo;all&raquo;_, _&laquo;single&raquo;_ and _&laquo;multi&raquo;_. Note that when the current mode is _&laquo;all&raquo;_, the **List** allows a&nbsp;user to&nbsp;select all items at&nbsp;once by&nbsp;clicking the _Select All_ check box. The **selectAllMode** property allows you to&nbsp;decide whether all items should be&nbsp;selected across all pages or&nbsp;within the currently rendered pages only.
\ No newline at end of file
+The **List** offers three item selection modes: *"single"*, *"multiple"*, and *"all"*.  To enable item selection, pass a mode name to the [selectionMode](/Documentation/ApiReference/UI_Components/dxList/Configuration/#selectionMode) property. If you set this property to *"none"* (default setting), item selection is disabled. In this example, you can try out all the selection modes. Use the Selection Mode drop-down menu to switch between them.
+
+List items can display checkboxes or radio buttons that help users select the items. To show these controls, enable the [showSelectionControls](/Documentation/ApiReference/UI_Components/dxList/Configuration/#showSelectionControls) property. Otherwise, users click or tap list items to select them. The **showSelectionControls** property must also be enabled when you use the *"all"* **selectionMode** so that the Select All checkbox is visible.
```

Comment:
```suggestion
List items can display checkboxes or radio buttons. To show these controls, enable the [showSelectionControls](/Documentation/ApiReference/UI_Components/dxList/Configuration/#showSelectionControls) property. Otherwise, users can click or tap list items to select them. The **showSelectionControls** property should also be enabled when you use the *"all"* **selectionMode** to show the Select All checkbox.
```